sacrebleu/setup.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+
+# Always prefer setuptools over distutils
+from setuptools import setup
+
+VERSION = '1.0.0'
+
+setup(
+    name = 'sacrebleu',
+
+    # Versions should comply with PEP440.  For a discussion on single-sourcing
+    # the version across setup.py and the project code, see
+    # https://packaging.python.org/en/latest/single_source_version.html
+    version = VERSION,
+
+    description = 'Hassle-free computation of shareable, comparable, and reproducible BLEU scores',
+    long_description = 'SacréBLEU is a standard BLEU implementation that downloads and manages WMT datasets, produces scores on detokenized outputs, and reports a string encapsulating BLEU parameters, facilitating the production of shareable, comparable BLEU scores.',
+
+    # The project's main homepage.
+    url = 'https://github.com/mjpost/sacreBLEU',
```

Comment:
You should update this URL and the other information in this file.

Reply:
Ah, missed that one, will do.